src/Gir/Model/Parameter.cs

```diff
@@ -43,6 +43,9 @@ public partial class Parameter
         [XmlElement("type")]
         public Type Type;
 
+        [XmlElement("varargs")]
+        public Varargs Varargs;
```

Comment:
This is just an empty tag so serializing this as a class does not really make sense. Maybe it's better to make it a private variable and class and rename it to VarargsClass. The public property can then be a boolean which just checks if VarargsClass is set.

Reply:
:+1: